dbt_subprojects/tokens/models/tokens/gnosis/tokens_gnosis_erc20.sql

```diff
@@ -7798,4 +7798,9 @@ FROM (VALUES
     , ( 0x4e5af8c1bd8ec1ffafa8a6043e57c3d384a7c574, 'ZTLN', 18)
     , ( 0x0deee1f9b96cac0b0ec85e8f800bc0b33e901e9a, 'IDOL', 18)
     , ( 0xfef5f9d8fe3b23fe4a242a461e537bb69364de94, 'VCUSD', 18)
-) AS temp_table (contract_address, symbol, decimals)
+    , (0xdd439304a77f54b1f7854751ac1169b279591ef7, 'EURe/sDAI', 18)
+    , (0xedbc7449a9b594ca4e053d9737ec5dc4cbccbfb2, 'aGnoEURe', 18)
+    , (0x06135a9ae830476d3a941bae9010b63732a055f4, 'stEUR/EURe', 18)
+    , (0xc791240d1f2def5938e2031364ff4ed887133c3d, 'rETH', 18)
+    , (0xf490c80aae5f2616d3e3bda2483e30c4cb21d1a0, 'osGNO', 18)
+) AS temp_table (contract_address, symbol, decimals)
```

Comment:
```suggestion
) AS temp_table (contract_address, symbol, decimals)

```

I guess the script doesn't add new lines :(